global.css

```diff
@@ -0,0 +1,544 @@
+
+@import url('css/bootstrap.css');
+
+/* @import url('desktop.css'); */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+ul li {
+  list-style: none;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  font-size: 1rem;
+}
```

Comment:
- [ ] Great work using Lato fonts for this family. For the project, you are to use both Cocogoose and Lato fonts. Kindly apply Cocogoose fonts for `hx` tags